src/blockchain.js

```diff
@@ -11,14 +12,67 @@ import {
   bitcoindSendRawTransaction,
   bitcoindParams,
   bitcoindGetAddressStatus,
+  isWalletAddressNotFoundError
 } from "./bitcoind";
 
 export const BLOCK_EXPLORER = 'public';
 export const BITCOIND = 'private';
 
+/**
+ * Fetch utxos for an address, calculate total balances
+ * and return an object describing the addresses state
+ * @param {string} address 
+ * @param {string} network 
+ * @param {object} client 
+ * @returns {object} slice object with information gathered for that address
+ */
 export async function fetchAddressUTXOs(address, network, client) {
-  const unsortedUTXOs = await fetchAddressUTXOsUnsorted(address, network, client);
-  return sortInputs(unsortedUTXOs);
+  let unsortedUTXOs, 
```

Comment:
The functionality implemented here was done in all three instances where this function was originally used complicating the implementation of those components quite a bit. I thought it made better sense to return the data in the format the app expects in any place it's running this call rather than give it raw and have to implement this data parsing each time we call `fetchAddressUTXOs`